swap_meet/decor.py

```diff
@@ -1,2 +1,12 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Decor(Item):
+    def __init__(self, id=None, condition=0, width=None, length=None):
+        super().__init__(id, condition)
+        self.width = 0 if width is None else width
+        self.length = 0 if length is None else length
```

Comment:
Instead of using `None` as the default values for `width` and `length`, we should use `0`, like the README outlines: "Holds 2 integer attributes width and length. Both of these values should be 0 by default"

When you use 0, then you don't need to have any additional logic when you set the instances width and length attribute.

```suggestion
    def __init__(self, id=None, condition=0, width=0, length=0):
        super().__init__(id, condition)
        self.width = width
        self.length = length
```